kit/azure/bootstrap/resources.deploy-spn.tf

```diff
@@ -104,6 +104,12 @@ resource "azuread_app_role_assignment" "cloudfoundation_deploy-directory" {
   resource_object_id  = data.azuread_service_principal.msgraph.object_id
 }
 
+resource "azuread_app_role_assignment" "cloudfoundation_deploy-group" {
```

Comment:
d: add a quick comment that this is necessary if you decide on managing groups from other kit modules. Productive use in a cloud foundation should probably manage groups not via terraform but via existing IAM processes, but this is a good lean start

Reply:
d: do we also need a new `resource_access` declarations on the `azuread_application` above?

Reply:
> d: do we also need a new `resource_access` declarations on the `azuread_application` above?

not sure, it worked without a new in my case.